autox/informer.go

```diff
@@ -3,154 +3,363 @@ package autox
 // informer.go - informer(s) to watch desired resources/namespaces
 
 import (
+	"bytes"
+	"context"
+	_ "embed"
+	"errors"
 	"fmt"
-	"reflect"
+	"sync"
 
+	"github.com/iter8-tools/iter8/base"
 	"github.com/iter8-tools/iter8/base/log"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/runtime/serializer/yaml"
 	"k8s.io/client-go/dynamic/dynamicinformer"
 	"k8s.io/client-go/tools/cache"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
-	autoxLabel = "iter8.tools/autox-group"
+	autoXLabel          = "iter8.tools/autox-group"
```

Comment:
@sriumcp I just realized that we are also using `autoXLabel` in a slightly different context here: https://github.com/Alan-Cha/iter8/blob/9ea003f729dd4f144aa77855942079a2a656ad1a/autox/informer.go#L82

I don't think it makes sense for it to be the `autoXLabel` in the secrets to be set to a boolean. I will create a new label called `autoXGroupLabel` for this use case then so that this label doesn't get mixed up with the `autoXLabel` in the trigger objects.